spring-batch-docs/asciidoc/common-patterns.adoc

```diff
@@ -78,7 +78,7 @@ public Step simpleStep() {
 }
 ----
 
-Remember that, if your listener does anything in an `onError()` method, it must be inside
```

Comment:
This is an important detail, so I think it's better emphasized in a note

Reply:
If it's really important, use the "IMPORTANT:" admonition. Otherwise, a "NOTE:" admonition will do.

Reply:
done, I changed it to IMPORTANT which is more appropriate indeed